bgpd/bgp_io.c

```diff
@@ -200,8 +200,8 @@ static int bgp_process_reads(struct thread *thread)
 	}
 
 	while (more) {
-		/* static buffer for transferring packets */
-		static unsigned char pktbuf[BGP_MAX_PACKET_SIZE];
+		/* buffer for transferring packets */
+		unsigned char pktbuf[peer->max_packet_size];
```

Comment:
This variable was static before but seems  it is removed and made it to allocate from stack?

Reply:
VLAs cannot be static. I think this is fine.

Reply:
> VLAs cannot be static. I think this is fine.

Right. I guess this is fine.